extensions/amp-story/1.0/amp-story-reaction-poll.css

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-reaction-poll-container {
+  display: flex !important;
+  justify-content: center !important;
+  height: 90px !important;
+  max-width: 296px !important;
+  border-radius: 16px !important;
+  box-shadow: 0px 2px 4px #000000aa !important;
+  color: var(--reaction-accent-color) !important;
+  background: var(--reaction-background-color) !important;
+  overflow: hidden !important;
+
+  --post-select-scale: 0.5 !important;
+  --post-select-scale-variable: 1 !important;
+}
+
+.i-amphtml-story-reaction-poll-container[theme="dark"] {
+  --reaction-accent-color: white;
+}
+
+.i-amphtml-story-reaction-poll-container:not(.i-amphtml-story-reaction-post-selection) {
+  cursor: pointer !important;
+}
+
+.i-amphtml-story-reaction-option {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+  text-align: center !important;
+  flex-direction: column !important;
+  flex-grow: 50 !important;
+  flex: 100 !important;
+  overflow: hidden !important;
+  position: relative;
+}
+
+.i-amphtml-story-reaction-poll-container:not(.i-amphtml-story-reaction-post-selection)
+  .i-amphtml-story-reaction-option-divider {
```

Comment:
Please add !important statements, here and in the following CSS block

Reply:
👍